### ISSUE TITLE
'Permission denied' when enabling events with events:on

### ISSUE THREAD

**Issue by bartolomej** [dokku/dokku#3637]
Dokku info
```
Client: Docker Engine - Community
        Version:           19.03.1
        API version:       1.40
        Go version:        go1.12.5
```

I was trying to enable dokku events logging and got this error.
```
root@app-server-1:~# dokku events:on
Enabling dokku events logger
/var/lib/dokku/plugins/enabled/20_events/subcommands/on: line 11: /home/dokku/.dokkurc/DOKKU_EVENTS: Permission denied
```

Also checked for permissions on that files and they seems to be ok.
```
-rwxr-xr-x 1 dokku dokku 403 Aug  7 11:33 /var/lib/dokku/plugins/enabled/20_events/subcommands/on
```

But noticed that DOKKU_EVENTS file doesn't exist
```
root@app-server-1:~# cat /home/dokku/.dokkurc/DOKKU_EVENTS
cat: /home/dokku/.dokkurc/DOKKU_EVENTS: No such file or directory
```

**Comment by josegonzalez** [comment]
Thats not all the `dokku report` output, but I think this is fixed by #3636.

**Comment by josegonzalez** [comment]
This should be fixed if you upgrade to 0.18.1.

**Comment by bartolomej** [comment]
Upgrading to the latest 0.18.2 fixed the problem, thank you !
